dev/com.ibm.ws.concurrency.policy/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -0,0 +1,45 @@
+###############################################################################
+# Copyright (c) 2017 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+# -------------------------------------------------------------------------------------------------
+#CMVCPATHNAME com.ibm.ws.concurrency.policy/resources/OSGI-INF/l10n/metatype.properties
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+
+# configuration elements
+
+concurrencyPolicy=Concurrency Policy
+concurrencyPolicy.desc=A set of behaviors and constraints to be applied for tasks that are capable of asynchronous execution. For example maximum concurrency and maximum queue size.
+
+# attributes
+
+expedite=Expedite
+expedite.desc=Specifies a core number of tasks to aim to run concurrently by expediting requests to the global thread pool. This provides no guarantee that this many tasks will run concurrently. If the expedite value is updated while in use, the update goes into effect gradually as previous expedited and non-expedited requests complete.
+
+max=Maximum concurrency
+max.desc=Specifies the maximum number of tasks that can be running at any given point in time. The default is unbounded, up to Integer.MAX_VALUE. Maximum concurrency can be updated while tasks are in progress. If the maximum concurrency is reduced below the number of concurrently executing tasks, the update goes into effect gradually as in-progress tasks complete rather than causing them to be canceled.
+
+maxPolicy=Maximum policy
+maxPolicy.desc=Indicates whether to loosely or strictly enforce maximum concurrency for tasks that run on the task submitter's thread. Tasks can run on the task submitter's thread when using the untimed invokeAll method, or, if only invoking a single task, the untimed invokeAny method. If the run-if-queue-full attribute is configured, it is also possible for tasks to run the task submitter's thread when using the execute and submit methods. In all of these cases, this attribute determines whether or not running on the submitter's thread counts against the maximum concurrency.
+maxPolicy.loose.desc=Maximum concurrency is loosely enforced, in that tasks are allowed to run on the task submitter's thread without counting against maximum concurrency.
```

Comment:
Updating maxPolicy.loose.desc as suggested.
However, the proposed change to say, "If you only invoke a single task, it uses the untimed invokeAny method," changes the meaning to something inaccurate rather than informing the customer of how untimed invokeAny can lead to a task running on the submitter's thread.  Need to propose a different change here that retains the original intent.